src/integrations/feed/configs.ts

```diff
@@ -111,3 +121,43 @@ export class FeedUserStateConfigGenerator implements FeedConfigGenerator {
     });
   }
 }
+
+type FeedLofnConfigGeneratorOptions = {
+  feed_version: FeedVersion;
+} & Options;
+
+export class FeedLofnConfigGenerator implements FeedConfigGenerator {
+  private readonly baseConfig: BaseConfig;
+  private readonly lofnClient: LofnClient;
+  private readonly opts: FeedLofnConfigGeneratorOptions;
+  private readonly feedPreferencesConfigGenerator: FeedPreferencesConfigGenerator;
+
+  constructor(baseConfig: BaseConfig, opts: FeedLofnConfigGeneratorOptions) {
+    this.baseConfig = baseConfig;
+    this.lofnClient = new LofnClient();
+    this.opts = opts;
+  }
+
+  async generate(
+    ctx: Context,
+    opts: DynamicConfig,
+  ): Promise<FeedConfigGeneratorResult> {
+    return runInSpan('FeedLofnConfigGenerator', async () => {
```

Comment:
If it ever becomes a bottleneck we can run `fetchConfig` and `FeedPreferencesConfigGenerator` in parallel. Probably will require changing a little bit `FeedPreferencesConfigGenerator`. We can consider it as part of the cleanup

Reply:
Yes, agreed 👍

Reply:
I'll take a look, it should not be too bad to implement it in this PR.

Reply:
Done 82511f866aaf8b1c58df9a9d1fb977e342e20b09